app/models/patient.rb

```diff
@@ -65,6 +69,7 @@ class Patient
             :primary_phone,
             :initial_call_date,
             :created_by,
+            :line,
```

Comment:
A perk of using `enum` here is that rails refuses to let this ever be null, so the validation is more for clarity's sake than anything, and as a consequence we don't need to bulk update any records. (Way to go us!) Here's counts from the prod db verifying counts:

```
irb(main):001:0> Patient.count
=> 670
irb(main):002:0> Patient.where(_line: :DC).count
=> 670
irb(main):003:0> Patient.where(_line: :MD).count
=> 0
irb(main):004:0> Patient.where(_line: :VA).count
=> 0
irb(main):005:0> Patient.where(_line: nil).count
=> 0
```